tests/conftest.py

```diff
@@ -1,40 +1,73 @@
-import os
-
 import pytest
+from unittest.mock import MagicMock
+
+from google.cloud.storage import Client
+from google.cloud.storage.bucket import Bucket
+from google.cloud.storage.blob import Blob
 
-from crypto.utils.api import Connection
-from crypto.utils.constants import LISTINGS_LATEST_URL
-from crypto.load import Load
-from crypto.transform import Transform
+
+@pytest.fixture
+def coinmarket_valid_response(mocker):
```

Comment:
mocker is a pytest plugin? if so when to use that one over Standard Library?

Reply:
as far as API and cloud service client mocking, this was my first real mocking experience. I leveraged GPT in some of these spots :). Feel like I recall watching one of the PDM training videos on mocking and there being discussion around this/similar topic/Standard Library one easier to work with than the pytest in some scenarios. 

I need to update the requirements.txt to reflect pytest-mock install.

Reply:
awesome, the mocking clinic went pretty well, I showed unit vs functional testing and mocked requests, sqlite and input()